dali/sasha/saxref.cpp

```diff
@@ -1402,6 +1410,7 @@ class CNewXRefManager: public CNewXRefManagerBase
         ndone[1] = 0;
         unsigned fnameHash = getFilenameHash(scopeBuf.str());
         const char * prefix = storagePlane->queryProp("@prefix");
+        bool isExternalFile = (size32_t)f->filenameLen == 0;
```

Comment:
for clarity, I think it would be good if cFileDesc had a isExternalFile() method.